RELEASE/scripts/sl_ascend.ash

```diff
@@ -14685,7 +14685,9 @@ void print_help_text()
 {
 	print_html("Thank you for using sl_ascend!");
 	print_html("If you need to configure or interrupt the script, choose <b>soolascend</b> from the drop-down \"run script\" menu in your browser.");
-	print_html("If you want to contribute, please open an issue at <a href=\"https://github.com/soolar/sl_ascend/issues\">on Github</a>");
+	print_html("If you want to contribute, please open an issue <a href=\"https://github.com/soolar/sl_ascend/issues\">on Github</a>");
+	print_html("A FAQ with common issues (and tips for a great bug report) <a href=\"https://docs.google.com/document/d/1AfyKDHSDl-fogGSeNXTwbC6A06BG-gTkXUAdUta9_Ns\">can be found here</a>");
+	print_html("The developers also hang around on the Ascension Speed Society discord.");
```

Comment:
Can you add the same link to the discord as is in README.md to this too? Might as well.